drivers/hydrophones/scripts/hydrophones

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
```

Comment:
Not sure if it's easy after you've committed ontop of it, but when you're commiting I think you can add '--author NAME'
I will review this PR on the plane to SD